src/components/DataTable/index.module.scss

```diff
@@ -0,0 +1,32 @@
+@use '../../pages/admin/quizzes/QuizList/index.module' as *;
+
+.formatTable {
+    width: 100%;
+    margin-top: 63px;
+  }
+
+.classCol{
+    @extend %style-table-head;
+  
+    &:nth-child(1) {
+      width: 153px;
+    }
+  
+    &:nth-child(2) {
+      width: 285px;
+    }
+  
+    &:nth-child(3) {
+      width: 517px;
+    }
+
+    &:nth-child(4) {
+        width: 151px;
+        text-align: center;
+    }
+
+    &:nth-child(5) {
+        width: 151px;
+        text-align: center;
```

Comment:
what would happen if there are more than 5 headers? please change approach.

maybe the header width can be styled in the component that calls the DataTable component